studiopress/simple-social-icons
# Add Reddit Icon

**Summary of change:**

Add Reddit Icon

**Have the changes in this PR been added to the documentation for this project?**

Yes

**How to test:**

1. Look for Reddit URI field in widget
2. Add a URI to the Reddit URI field
3. Check that the Reddit icon has appeared
